specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2019-08-08/storageaccounts.json

```diff
@@ -451,6 +428,21 @@
           "description": "Storage account supports https traffic only or not",
           "type": "boolean",
           "readOnly": true
+        },
+        "accessTier": {
+          "description": "Required for storage accounts where kind = BlobStorage.",
+          "$ref": "#/definitions/StorageAccountAccessTier",
+          "readOnly": true
+        },
+        "overQuota": {
```

Comment:
Can it be overQuota AND suspended?  Using an enum (modeled as string) instead of multiple booleans to represent state is more flexible to future changes.  Otherwise, you can end up with many individual boolean parameters.  If it can be both it may make sense to have an enum for quota related state and another for account level state (i.e. suspended) depending on what overQuota and suspended actually mean

Reply:
I'll change it into an enumerator.